### PR TITLE
Add offset#fetchCommitsV1 functionality and tests

### PR DESCRIPTION
**Pull request by danjwinter**
References #558, #954

offset#fetchCommits gets the latest committed offsets from zookeeper. This pull request adds offset#fetchCommitsV1 functionality for fetching commits from the brokers, which aligns with how ConsumerGroup commits are stored.

Current implementation uses the same data structure as an input that offset#fetchCommits uses and converts it to the V1 format.

### CODE REVIEW

File: test/test.offset.js [hunk 1]
```diff
@@ -146,3 +186,33 @@ describe('Offset', function () {
...
+    var consumerGroupOptions = {
+      groupId: groupId,
+      fromOffset: 'earliest',
+      kafkaHost: 'localhost:9092',
+      autoCommitIntervalMs: 1,
+      autoCommit: true
+    };
+    var consumerGroup = new ConsumerGroup(consumerGroupOptions, topic);
+    consumerGroup.on('message', (message) => {
+      console.log('got message');
```
Comment (hyperlink) [inline test/test.offset.js]: can we remove these `console.log` ?

File: test/test.offset.js [hunk 2]
```diff
@@ -105,6 +106,45 @@ describe('Offset', function () {
     });
   });
 
+  describe('#fetchCommitsV1', function () {
+    var topic, topics, groupId, expectedCommittedOffset;
+    topic = `_exist_topic_3_test`;
+    topics = [ { topic: topic, partition: 0 } ];
+    groupId = `_groupId_commit_v1_test`;
+    before(function (done) {
+      producer.send([{ topic, messages: ['firstMessage'] }], (err, data) => { console.log(`Producer sent data: ${JSON.stringify(data)}, err: ${JSON.stringify(err)}`); });
```
Comment (hyperlink) [inline test/test.offset.js]: please remove this `console.log`.

**Comment by danjwinter** [comment]
Thanks for your comments @hyperlink! I made the requested changes, let me know if anything else needs to be adjusted.

**Comment by hyperlink** [comment]
@chrisandrews7 I will post a message here when it gets published.

**Comment by hyperlink** [comment]
Published as 3.0.0

**Pull request merged**